<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Knight's move</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#GameWindow {
	background-color: rgb(255, 255, 255);
	border: 1px solid #CCCCCC;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>650</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>650</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget {
	background-color: rgb(255, 255, 255);
	border: 1px solid #CCCCCC;
}</string>
   </property>
   <widget class="QTableView" name="gameField">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>500</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#gameField {
  border-top: 1 solid #cccccc;
  border-right: 1 solid #cccccc;
  border-bottom:  1 solid #cccccc;
  border-left: 1  solid #cccccc;
}

#gameField::item {
  border : 0px;
  padding-left: 0px;
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QLabel" name="headerLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>300</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Игра &quot;Ход конём&quot;</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pbStartGame">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>70</y>
      <width>330</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pbStartGame {
    background-color: #9100da;
    border-color: #9100da;
    border-style: solid;
    color: #ffffff;
    border-width: 1px;
}

#pbStartGame:hover {
    background-color: #7e00c2;
}

#pbStartGame:pressed {
    background-color: #63009d;
}</string>
    </property>
    <property name="text">
     <string>Начать игру</string>
    </property>
   </widget>
   <widget class="QTableView" name="verticalDigitsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>50</width>
      <height>500</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>500</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>500</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#verticalDigitsView {
  border-top: 1 solid #cccccc;
  border-right: 1 solid #cccccc;
  border-bottom:  1 solid #cccccc;
  border-left: 1  solid #cccccc;
}

#verticalDigitsView::item {
  border : 1px;
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
   </widget>
   <widget class="QTableView" name="horizontalLettersView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>580</y>
      <width>500</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>500</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#horizontalLettersView {
  border-top: 1 solid #cccccc;
  border-right: 1 solid #cccccc;
  border-bottom:  1 solid #cccccc;
  border-left: 1  solid #cccccc;
}

#horizontalLettersView::item {
  border : 1px;
  padding-left: 0px;
}</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>50</number>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pbCancelOneMove">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>330</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pbCancelOneMove {
    background-color: #ec00ec;
    border-color: #ec00ec;
    border-style: solid;
    color: #ffffff;
    border-width: 1px;
}

#pbCancelOneMove:hover {
    background-color: #d400d4;
}

#pbCancelOneMove:pressed {
    background-color: #a800a8;
}

#pbCancelOneMove:disabled {
    background-color: #bdbdbd;
	border-color: #bdbdbd;
}</string>
    </property>
    <property name="text">
     <string>Отменить один ход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbCancelMoves">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>330</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pbCancelMoves {
    background-color: #ec00ec;
    border-color: #ec00ec;
    border-style: solid;
    color: #ffffff;
    border-width: 1px;
}

#pbCancelMoves:hover {
    background-color: #d400d4;
}

#pbCancelMoves:pressed {
    background-color: #a800a8;
}

#pbCancelMoves:disabled {
    background-color: #bdbdbd;
	border-color: #bdbdbd;
}</string>
    </property>
    <property name="text">
     <string>Отменить ходы</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbExit">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>580</y>
      <width>330</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pbExit {
    background-color: #9100da;
    border-color: #9100da;
    border-style: solid;
    color: #ffffff;
    border-width: 1px;
}

#pbExit:hover {
    background-color: #7e00c2;
}

#pbExit:pressed {
    background-color: #63009d;
}</string>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leMovesNumber">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>370</y>
      <width>330</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>330</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>330</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
